client/my-sites/customer-home/cards/tasks/task.jsx

```diff
@@ -148,13 +148,13 @@ const Task = ( {
 							className="task__skip-popover"
 						>
 							<PopoverMenuItem onClick={ () => skipTask( '1d' ) }>
-								{ translate( 'Tomorrow' ) }
+								{ translate( 'For a day' ) }
 							</PopoverMenuItem>
 							<PopoverMenuItem onClick={ () => skipTask( '1w' ) }>
-								{ translate( 'Next week' ) }
+								{ translate( 'For a week' ) }
 							</PopoverMenuItem>
 							<PopoverMenuItem onClick={ () => skipTask() }>
-								{ translate( 'Never' ) }
+								{ translate( 'Forever' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'free forever' )` *ES Score: 7*